player.js

```diff
@@ -33,10 +34,12 @@ if (argv.q || argv.quiet) {
 const allContents = JSON.parse(fs.readFileSync('./.data/allContents.json'))
 const playlist = []
 
+const screenResolution = sr.get()
```

Comment:
`const [ width, height ] = sr.get()` is more explicit (and this way we can call the required module `screenres` or `screenResolution` without worrying about confusing names)